### PR TITLE
Resource filtering

### PR DESCRIPTION
**Pull request by R-Lawton**

### CODE REVIEW

**Comment by R-Lawton** [comment]
CC: @jasonmadigan

File: locales/en/plugin__kuadrant-console-plugin.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
   "Reference to an existing secret resource containing DNS provider credentials and configuration": "Reference to an existing secret resource containing DNS provider credentials and configuration",
   "Release Notes": "Release Notes",
   "Save": "Save",
+  "Search across name, namespace and/or resource type": "Search across name, namespace and/or resource type",
```
Comment (jasonmadigan) [inline locales/en/plugin__kuadrant-console-plugin.json:88]: Wonder could we shorten this to fit the input?

maybe:
"Search by name, namespace or type"

File: src/components/ResourceList.tsx [hunk 2]
```diff
@@ -490,7 +530,17 @@ const ResourceList: React.FC<ResourceListProps> = ({
...
         <ListPageBody>
-          <ListPageFilter data={data} loaded={allLoaded} onFilterChange={onFilterChange} />
+          <div className="kuadrant-filter-bar" style={{ marginBottom: '16px', marginTop: '15px' }}>
+            <SearchInput
+              placeholder={t('Search across name, namespace and/or resource type')}
+              value={String(filters) || ''}
+              onChange={(_, value: string) => handleFilterChange(value)}
+              onClear={handleClear}
+              aria-label="Global search input"
+              style={{ marginRight: '8px', width: '350px' }}
```
Comment (jasonmadigan) [inline src/components/ResourceList.tsx:540]: nit: move to class

Comment (jasonmadigan) [reply src/components/ResourceList.tsx:540]: worky pretty good. Wonder if we should look at a toolbar instead in keeping with other resource views?

https://www.patternfly.org/components/toolbar/html-demos/#toolbar-attribute-value-search-filter-on-mobile
Screenshot 2024-11-19 at 10 49 36

**Comment by R-Lawton** [comment]
fixed a issue with dark mode where a diff shade of gray was showing when there wasn't as many resources showing in the table 
Screenshot 2024-11-19 at 17 10 22
